guardian/newsletters
# Rachel/add regional focus

## What does this change?

Add field to version 0.8 of newsletters spreadsheet & API to indicate the regional focus of each newsletter.  The field has the title "region focus" and is located between the existing "description" and "identity name" fields.

Where a newsletter doesn't have a regional focus, the field can be left blank.

## How to test

Run the tests locally using yarn test
Use yarn preview to check that the JSON file contains the new region focus field
